src/config.js

```diff
@@ -55,6 +59,7 @@ class ReduxEntityConfig extends BaseConfig {
             ...state,
             loading: false,
             errors: {},
+            sortedIds: _without(state.sortedIds, payload.data),
```

Comment:
Yup, I think it should.

Reply:
For posterity: no change to the `sortedIds` array for a deletion, since there's ambiguity as to whether an object will be returned with the API response.